arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/extensions/ior/apply/IorApply.kt

```diff
@@ -71,9 +71,7 @@ fun <L, A, B> Kind<Kind<ForIor, L>, A>.apEval(
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-    "this.zipEval<L, A, B, Z>(SL, arg1) { a, b -> arg2(Tuple2(a, b)) }",
-    "arrow.core.zipEval",
-    "arrow.core.Tuple2"
+    "arg1.map { zip(it) { a, b -> arg2(Tuple2(a, b)) } }", "arrow.core.Tuple2"
```

Comment:
maybe missing import for `zip`?
```suggestion
    "arg1.map { zip(it) { a, b -> arg2(a, b) } }", "arrow.core.Tuple2", "arrow.core.zip"
```